app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -40,22 +50,25 @@ document.addEventListener('turbolinks:load', function() {
         case 'skip':
           $fieldLabel.text('N/A');
           break;
-        default: 
+        default:
           var header = $fieldLabel.data('header');
           $fieldLabel.text(header);
       }
 
       _setDradisFieldSelect($(this));
     });
 
-    $('[data-behavior~=mapping-form]').submit(function() {
+    $('[data-behavior~=mapping-form]').submit(function () {
       var valid = _validateIdentifierSelected() && _validateNodeSelected();
 
       if (!valid) {
-        $(this).find('input[type="submit"]').attr('disabled', false).val('Import CSV');
+        $(this)
+          .find('input[type="submit"]')
+          .attr('disabled', false)
+          .val('Import CSV');
 
         $('[data-behavior~=view-content]').animate({
-          scrollTop: $('[data-behavior~=validation-messages]').scrollTop()
+          scrollTop: $('[data-behavior~=validation-messages]').scrollTop(),
```

Comment:
Is the added comma necessary?

Reply:
This was the linter adding it in

Reply:
In that case, this can be removed in the release branch